pkg/storage-init/storage-init.sh

```diff
@@ -13,6 +13,9 @@ if CONFIG=$(/hostfs/sbin/findfs PARTLABEL=CONFIG) && [ -n "$CONFIG" ]; then
     if ! mount -t vfat -o dirsync,noatime "$CONFIG" $CONFIGDIR; then
         echo "$(date -Ins -u) mount $CONFIG failed"
     fi
+    if ! fsck.vfat -y "$CONFIG"; then
```

Comment:
Doing a fsck -y after the mount seems wrong. Shouldn't it be done first?

Reply:
My bad. Copy paste error from a different sandbox. Will fix it.